theano/sandbox/gpuarray/elemwise.py

```diff
@@ -204,60 +173,64 @@ def generate_kernel(self, node, nodename):
             kop = kop.replace(npy, ga)
         return ElemwiseKernel(None, inps+outs, kop, preamble=support_code)
 
-    def c_headers(self):
+    def c_header_dirs(self):
         if pygpu.get_default_context().kind == 'opencl':
             raise MethodNotDefined('cuda only')
-        return ['cuda.h', '<gpuarray/extension.h>', '<numpy_compat.h>',
-                '<gpuarray/ext_cuda.h>']
+        cuda_root = config.cuda.root
+        if cuda_root:
+            return [os.path.join(cuda_root, 'include')]
+        else:
+            return []
 
     def c_compiler(self):
+        return GCC_compiler
```

Comment:
Without it, GpuIncSubtensor.c_compiler() fails with

```
AttributeError: ('The following error happened while compiling the node', GpuIncSubtensor{InplaceSet;:int64:}(GpuAlloc.0, Rebroadcast{0}.0, Constant{1}), '\n', "'GpuElemwise' object has no attribute 'c_compiler'")
```
as it is defined as
```
def c_compiler(self):
    return self.iadd_node.op.c_compiler()
```

So, I think I still need this.  Do you think I am missing something?